### PR TITLE
Master data load

### PR DESCRIPTION
**Pull request by MonishaSivanathan**
- added all the current data in the load file

### CODE REVIEW

**Comment by tamsud** [comment]
Add rxnorm master data as well as in the script

**Comment by MonishaSivanathan** [comment]
added icd, snomed and rxnorm

File: scripts/load_master_data.py [hunk 1]
```diff
@@ -90,57 +98,85 @@
...
+    {"display": "Progressive Insurance", "code": "24260"},
+    {"display": "State Farm Insurance", "code": "25178"},
+    {"display": "The General Insurance", "code": "13703"},
+    {"display": "Travelers Insurance", "code": "25658"},
+    {"display": "USAA Insurance", "code": "186000"},
+]
+
+concept_data = [
+    {
+        "url" : "https://storage.googleapis.com/ftr/icd10cm/vs/http%3A--hl7.org-fhir-ValueSet-icd-10/tf.5e7b9e0dc05aace8f1a3c8b086556438b0b87848c9304fe9999d8b237e67a8ee.ndjson.gz",
```
Comment (tamsud) [inline scripts/load_master_data.py:172]: will dict allow duplicate keys?
Have you done testing?

File: scripts/load_master_data.py [hunk 2]
```diff
@@ -7,16 +7,20 @@
...
 # Table creation data
 tables = [
-    #{"id": "CimparRace", "type": "resource", "isOpen": True},
+    {"id": "CimparRace", "type": "resource", "isOpen": True},
     {"id": "CimparEthnicity", "type": "resource", "isOpen": True},
     {"id": "CimparState", "type": "resource", "isOpen": True},
-    {"id": "CimparLabTest", "type": "resource", "isOpen": True}
+    {"id": "CimparLabTest", "type": "resource", "isOpen": True},
+    {"id": "CimparInsuranceCompany", "type": "resource", "isOpen": True},
+    {"id": "Concept"}
```
Comment (tamsud) [inline scripts/load_master_data.py:23]: is that necessary to create the concept table?

**Review by tamsud — CHANGES_REQUESTED** [review]

File: scripts/load_master_data.py (hunk 1 above)
Comment (MonishaSivanathan) [reply scripts/load_master_data.py:172]: Screenshot from 2024-08-23 12-42-24

Comment (MonishaSivanathan) [reply scripts/load_master_data.py:172]: i have tested, the data is added

File: scripts/load_master_data.py (hunk 2 above)
Comment (MonishaSivanathan) [reply scripts/load_master_data.py:23]: ti check the condition alone i have added

**Comment by tamsud** [comment]
why duplicate keys are there in the dictionary

**Review by tamsud — APPROVED** [review]

**Pull request merged**